manim/utils/ipython_magic.py

```diff
@@ -137,12 +137,5 @@ def construct(self):
                 )
 
 
-def _video_hash(path):
-    sha1 = hashlib.sha1()
-    with open(path, "rb") as f:
-        while True:
-            data = f.read(65536)
-            if not data:
-                break
-            sha1.update(data)
-    return sha1.hexdigest()
+def _generate_file_name():
+    return config["scene_names"][0] + "@" + datetime.now().strftime("%y-%m-%d@%H-%M-%S")
```

Comment:
```suggestion
    return config["scene_names"][0] + "@" + datetime.now().strftime("%Y-%m-%d@%H-%M-%S")
```

Reply:
I see your argument, now I also like the 4 digit approach!
Just click on "Commit suggestion", and the 4 digit formatting is back :)
I think after this, it can get merged.